### DISCUSSION TITLE
Permanent Highlight for Forward Search in WinEdtPDF

### DISCUSSION THREAD

**Discussion by kwokkun** [WinEdt-Team/WinEdt-Support#58] [1.3 Ideas and features requests]
There's an option called "HighlightPermanent" in "ForwardSearch" in SumatraPDF, which enables the highlight to be permanent. I was wondering if the same can be implemented in WinEdtPDF. Thanks!

**Comment by Alex-Simonic** [comment]
This could be easily done in future versions... Not many users will care about it but then again who am I to judge what users may or may not want?

**Reply by Alex-Simonic** [reply]
In the next build, you will have an option for "Persistent Forward Search Highlighting" in the Preferences| Advanced Dialog. You can use a mouse click or ESC Key to remove the "persistent" highlighting. That should address this issue...
